src/vcpkg/binaryparagraph.cpp

```diff
@@ -280,16 +280,10 @@ namespace vcpkg
             out_str.substr(initial_end), "vcpkg::serialize(const BinaryParagraph&, std::string&)");
         if (!parsed_paragraph)
         {
-            Checks::exit_maybe_upgrade(VCPKG_LINE_INFO,
-                                       R"([sanity check] Failed to parse a serialized binary paragraph.
-Please open an issue at https://github.com/microsoft/vcpkg, with the following output:
-    Error: %s
-
-=== Serialized BinaryParagraph ===
-%s
-            )",
-                                       parsed_paragraph.error(),
-                                       my_paragraph);
+            Checks::msg_exit_maybe_upgrade(
```

Comment:
`[sanity check]` makes it sound like a debugging message, but it clearly has user instructions, so it makes sense.

That said, it seems like line 293 should get the same treatment?

Reply:
> `[sanity check]` makes it sound like a debugging message, but it clearly has user instructions, so it makes sense.
> 
> That said, it seems like line 293 should get the same treatment?

Please check out the related change. :)